plugins/home/src/api/VisitsApiFactory.test.ts

```diff
@@ -0,0 +1,323 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Visit } from './VisitsApi';
+import { VisitsApiFactory } from './VisitsApiFactory';
+
+class MemoryVisitsApi extends VisitsApiFactory {
```

Comment:
Suggestion, totally fine for a separate pull request.
This sounds like a good API mock test utils candidate 💡 .
I would recommend exporting it from the package on from the test-utils package as the Analytics plugin did:
https://github.com/backstage/backstage/blob/f4abd00ac2137aaac6be25b81a8b9731d0e73305/packages/test-utils/src/testUtils/apis/AnalyticsApi/MockAnalyticsApi.ts#L25

Reply:
Good point. I can follow it up in another PR.